### ISSUE TITLE
sort by similarity? 

### ISSUE THREAD

**Issue by CuddleBear92** [hydrusnetwork/hydrus#88]
sort by similarity? 
not sure if this can even be done.
would be nice to have a quick way to sort similar images and so on when they don't have a title/issue/page and so on to sort after.
yes you can argue that you can sort by filename. but that is only when you actually have a file name to search after, doesnt help that files on boorus doesnt contain filenames either and you cant import folders with the filename namespace either.

**Comment by CuddleBear92** [comment]
Hey, issues here have been reenabled and will be used for bug reports/feature requests in the future.

We are currently clearing out all the existing issues from 2017 and prior.
If your bug report or feature request is still relevant, please create a new issue (or use one of the other channels; see https://hydrusnetwork.github.io/hydrus/).

**Comment by CuddleBear92** [comment]
Hey, issues here have been reenabled and will be used for bug reports/feature requests in the future.

We are currently clearing out all the existing issues from 2017 and prior.
If your bug report or feature request is still relevant, please create a new issue (or use one of the other channels; see https://hydrusnetwork.github.io/hydrus/).

**Comment by CuddleBear92** [comment]
Re-opening as this issue still stands. Cleaning up comments.

**Comment by bbappserver** [comment]
@CuddleBear92 Similarity is not a sortable value because `a ~= b` == `b ~= a`.  Did you mean group by? And if so what is wrong with the duplicates system at present for this?

**Comment by CuddleBear92** [comment]
No, sort. I still stand by my word since 2015. Sort orders by dupe id/group.
That way it can more easily be sorted with dupes next to eachother in a normal gallery view.

Its a different usecase than the dupe tab itself.

**Comment by bbappserver** [comment]
Well similarity is a metric between two images not a metric on each image that can be used to compare A and B, so you would have to sort on similarity relative to a particular image.

It sounds like what you want is for already duplicate processed groups where it was determined that a king of those groups should be next to its subordinates, and that's fine unless you have a weird edge case where one of its subordinates is also king of a different group, in which case there goes your ordering out the window.

**Comment by DonaldTsang** [comment]
Imagine applying this to things that have no standard sizes like music, videos, or even ebooks... but it can be done with hashes and summarization.
